includes/definitions/routeros.yaml

```diff
@@ -13,3 +13,8 @@ discovery:
     -
         sysObjectID:
             - .1.3.6.1.4.1.14988.1
+oids:
+    unordered:
+        - IP-FORWARD-MIB::ipCidrRouteEntry
```

Comment:
It is better to use OIDs called Table instead of Entry.  Can you please change this and the related walk to `IP-FORWARD-MIB::ipCidrRouteTable`

Reply:
> It is better to use OIDs called Table instead of Entry. Can you please change this and the related walk to `IP-FORWARD-MIB::ipCidrRouteTable`

done
yes, you are right, need to get used to it table/entry